### PR TITLE
parser/opcode: fix format function for mod opcode

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

Before this fix, format `Mod` opcode will get `%!(NOVERB)`

### What is changed and how it works?

Use fmt.Printf correctly.

### Check List 

Tests 

 - Unit test

@lysu @jackysp

### CODE REVIEW

File: parser/opcode/opcode.go [hunk 1]
```diff
@@ -134,5 +134,5 @@ var opsLiteral = map[Op]string{
 
 // Format the ExprNode into a Writer.
 func (o Op) Format(w io.Writer) {
-	fmt.Fprintf(w, opsLiteral[o])
+	fmt.Fprintf(w, "%s", opsLiteral[o])
```
Comment (lysu) [inline parser/opcode/opcode.go]: maybe `w.Write([]byte(opsLiteral[o]))` is better? 0.0?

Comment (lysu) [reply parser/opcode/opcode.go]: or make `opsLiteral` be `map[Op][]byte` then `w.write(opsLiteral[o])`

Comment (tiancaiamao) [reply parser/opcode/opcode.go]: There is still a memory allocation in `w.Write([]byte(opsLiteral[o]))` when converting string to []byte.
If you want better performance, the 'hack' could be used, but I don't think it matters here.

**Pull request merged**